app/controllers/application_controller.rb

```diff
@@ -1,2 +1,6 @@
 class ApplicationController < ActionController::Base
+  before_action :authenticate_user!
+  def current_user
+    User.first
+  end
```

Comment:
- [ ] Please delete the `current_user` method in `ApplicationController` since you have installed `device`